src/Commands/MakeDataTransferObjectCommand.php

```diff
@@ -2,11 +2,13 @@
 
 namespace Regnerisch\LaravelBeyond\Commands;
 
+use Regnerisch\LaravelBeyond\Resolvers\AppNameSchemaResolver;
 use Regnerisch\LaravelBeyond\Resolvers\DomainNameSchemaResolver;
+use Illuminate\Support\Str;
 
 class MakeDataTransferObjectCommand extends BaseCommand
 {
-    protected $signature = 'beyond:make:dto {name?} {--overwrite}';
+    protected $signature = 'beyond:make:dto {name?} {--overwrite} {--factory}';
```

Comment:
Please remove the flag again, I do not like how the flag is handled. I know this is currently the only possible solution but I will develop a better way in v4. So for now we will drop the `factory`-flag and add it again in v4.